spec/e2e/query_proxy_methods_spec.rb

```diff
@@ -72,6 +72,110 @@ def destroy_called
     end
   end
 
+  describe 'find_or_initialize_by' do
+    context 'with relations chain' do
```

Comment:
I would say that this should be a `describe`, not `context`.  `context` generally talks about the `let`/`before` things that you're setting up, where as here you`re further specifying that you're describing using `find_or_initialize_by` with relations chains